example/macros/json.dart

```diff
@@ -5,14 +5,14 @@ const jsonSerializable = _JsonMacro();
 class _JsonMacro implements ClassDeclarationMacro, ClassDefinitionMacro {
   const _JsonMacro();
 
-  void declare(TargetClassDeclaration declaration) {
+  void forClassDeclaration(TargetClassDeclaration declaration) {
```

Comment:
How about `declareClass`, `defineClass`, `defineField`, etc? The `for` seems a little arbitrary to me.

Reply:
Well, it sounds like I am declaring an actual class without the `For`. I definitely am struggling with the names though.